README.md

```diff
@@ -14,7 +14,12 @@ Dependencies
 -   [**mraa**](https://github.com/intel-iot-devkit/mraa): This is not on PyPI and require additonal install steps. Follow the install instruction on [sparkfun](https://learn.sparkfun.com/tutorials/installing-libmraa-on-ubilinux-for-edison) with a couple updates:
    -   The step `apt-get install python-dev` needs to be updated to `apt-get install python3-dev`
    -   The step `cmake .. -DBUILDSWIGNODE=OFF` needs to be updated to `cmake .. -DBUILDSWIGNODE=OFF -DBUILDPYTHON3=ON`
-   -   If `export PYTHONPATH=$PYTHONPATH:$(dirname $(find /usr/local -name mraa.py))` doesn't work then `export PYTHONPATH=$PYTHONPATH:/usr/local/lib/python3.4/site-packages` should suffice`
+   -   When appending the python path in .bashrc as outlined in the instructions, there is a syntax error: `export PYTHONPATH=$PYTHONPATH:$(dirname $(find /usr/local -name mraa.py))` should have double quotes inside it, like so: `export PYTHONPATH=$PYTHONPATH:$(dirname $"(find /usr/local -name mraa.py)")` If that doesn't work, then `export PYTHONPATH=$PYTHONPATH:/usr/local/lib/python3.4/site-packages` should suffice.
```

Comment:
There is also a dependencies section for the Write block. All this info needs to be there too.